litellm/llms/astra_assistants.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, Union
+from typing_extensions import override
+import httpx
+from openai import OpenAI
+from ..llms.openai import OpenAIAssistantsAPI
+from astra_assistants import patch
+import astra_assistants
+
+
+
+class AstraAssistantsAPI(OpenAIAssistantsAPI):
```

Comment:
i don't see why this needs to exist as a separate file / if-else block in all the assistant api endpoints. 

If it's openai-compatible, all you need to do is update `get_llm_provider` like we do for groq 

and this should work without major changes. 

See - https://github.com/BerriAI/litellm/pull/2168/files#diff-9661b0d8f1d9f48433f2323d1102e963d3098f667f7a25caea79f16fa282f6dd

Reply:
Oh cool, I didn't realize that thanks!